addons/mail/sfu_server/src/shared/enums.js

```diff
@@ -0,0 +1,46 @@
+// separate file to avoid circular dependencies or increasing bundle size (despite tree shaking)
+// it also allows documentation to be shared between client and server
+
+export const WS_CLOSE_CODE = {
+    CLEAN: 1000,
+    LEAVING: 1001,
+    ERROR: 1011,
+    // 4000-4999 range available for app specific use
+    AUTHENTICATION_FAILED: 4106,
+    TIMEOUT: 4107,
+    KICKED: 4108,
+};
+
+export const SERVER_REQUEST = {
+    /** Requests the creation of a consumer that is used to forward a track to the client */
+    INIT_CONSUMER: "INIT_CONSUMER",
+    /** Requests the creation of upload and download transports */
+    INIT_TRANSPORTS: "INIT_TRANSPORTS",
+    /** Requests any response to keep the session alive */
+    PING: "PING",
+};
+
+export const SERVER_MESSAGE = {
+    /** Signals the clients that one of the session in their channel has left. */
+    SESSION_LEAVE: "SESSION_LEAVE",
+    /**  Signals the clients that the info (talking, mute,...) of one of the session in their channel has changed. */
+    INFO_CHANGE: "S_INFO_CHANGE",
+};
+
+export const CLIENT_REQUEST = {
+    /** Requests the server to connect the client-to-server transport, this occurs the first time a producer is added to this transport */
+    CONNECT_CTS_TRANSPORT: "CONNECT_CTS_TRANSPORT",
+    /** Requests the server to connect the server-to-client transport, this occurs the first time a consumer is added to this transport */
+    CONNECT_STC_TRANSPORT: "CONNECT_STC_TRANSPORT",
```

Comment:
Yes both transports are initialized by the server (with [`router.createWebRtcTransport()`](https://mediasoup.org/documentation/v3/mediasoup/api/#router-createWebRtcTransport)), but the "connect" is related to the ["connect" event](https://mediasoup.org/documentation/v3/mediasoup-client/api/#transport-on-connect) on the client-side transports.

Reply:
Seems weird that this would involve user code instead of being handled internally by mediasoup but I guess it's just a quirk of the mediasoup API like the need for extra signaling for production changes.